features/htcondor/templ/condor_mic_discovery.pan

```diff
@@ -0,0 +1,27 @@
+structure template features/htcondor/templ/condor_mic_discovery;
+
+'text' = {
+  txt = <<EOF;
+#! /usr/bin/perl
+
+use strict;
+use warnings;
+
+open(MICCTRL,'micctrl -s|') or die 'cannot run micctrl command';
+
+my @cards=();
+
+while(<MICCTRL>){
+  if(/^(\S+):\s*online\s*/){
+    push(@cards,$1);
+  }
+}
+
+close MICCTRL;
+
+print 'DetectedPhis="'.join(', ',@cards).'"'."\n";
```

Comment:
* Indentation should be a multiple of four spaces

Reply:
I've fixed the indentation on this one (but I'm a bit confused as there is no  indentation in the lines you have outlined)

Reply:
I've realized that actually this file is obsolete and never used in the configuration.... I've removed it and slightly changed features/htcondor/config to allow more flexibility in the configuartion (so now the perl script is in a separate file).